src/Query/Group.ts

```diff
@@ -1,13 +1,13 @@
 import type { Task } from '../Task';
 import { Priority } from '../Task';
-import type { Grouping, GroupingProperty } from './Query';
+import type { Grouping, GroupingArg, GroupingProperty } from './Query';
```

Comment:
I am soon going to be refactoring the grouping code to remove `GroupingProperty`.

I suggest we work on getting this PR merged first, to avoid this otherwise being blocked on a non-trivial refactoring.